.openshiftio/application.yaml

```diff
@@ -6,6 +6,14 @@ metadata:
     description: This template creates a Build Configuration using an S2I builder.
     tags: instant-app
 parameters:
+  - name: SUFFIX_NAME
+    description: The suffix name for the template objects
+    displayName: Suffix name
+    value: ''
```

Comment:
i think this should actually be `latest` or would it be `:latest`.  I noticed in the labels below the value is `nodejs-health-check${SUFFIX_NAME}` with no `:`, so would the value of this parameter need to add the `:`, so the resulting label would be `nodejs-health-check:some_value` instead of `nodejs-health-checksome_value`

Reply:
Hey @lholmquist 

In the orignal file, this parameter was not there, and we introduced this new variable and used as a suffix in the name of all resources. I kept the default value to empty string because someone is not specifying this parameter, so it will pick the default value and will be the case of your use case like previously.